pkg/systemd/hw-detect.js

```diff
@@ -120,6 +120,16 @@ function findMemoryDevices(udevdb, info) {
     info.memory = memoryArray;
 }
 
+async function getBootType() {
+    const secure_boot_file = cockpit.manifests.system.config.secure_boot_file;
```

Comment:
Why the indirection?  Storing this in the manifest seems very strange to me.

Reply:
Wasn't able to create the folder structure for `/sys/firmware/efi/efivars`

Reply:
I mean you could just hardcode it directly in the JS...  This is a well-known value.

Reply:
Yes, please do -- this isn't configuration.